pkg/apis/org/v1/che_types.go

```diff
@@ -138,6 +138,15 @@ type CheClusterSpecServer struct {
 	// Disabling TLS may cause malfunction of some Che components.
 	// +optional
 	TlsSupport bool `json:"tlsSupport"`
+
+	// Use internal host names provided by cluster dns.
+	// It should be usefull to speed internal services traffic.
+	// Also it could avoid some proxy issues.
+	ServiceHostnames bool `json:"serviceHostnames"`
```

Comment:
What is the expected default value for this field ?
If it's `true`, I assume that the Go type should be `*bool` instead of `bool` ?